src/components/CreatePostInput/index.js

```diff
@@ -249,4 +251,16 @@ const styles = StyleSheet.create({
   }
 });
 
+CreatePostInput.propTypes = {
+  allTaggedUser: PropTypes.array,
+  topics: PropTypes.array,
+  topicChats: PropTypes.array,
+  setAllTaggedUser: PropTypes.func,
+  setPositionKeyboard: PropTypes.func,
+  setTopics: PropTypes.func,
+  setTopicChats: PropTypes.func,
+  message: PropTypes.string,
+  setMessage: PropTypes.func
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding PropTypes validation is a good practice for ensuring that the correct types of props are passed to the component. This can help catch bugs related to incorrect prop types.
```diff
+ CreatePostInput.propTypes = {
+   allTaggedUser: PropTypes.array,
+   topics: PropTypes.array,
+   topicChats: PropTypes.array,
+   setAllTaggedUser: PropTypes.func,
+   setPositionKeyboard: PropTypes.func,
+   setTopics: PropTypes.func,
+   setTopicChats: PropTypes.func,
+   message: PropTypes.string,
+   setMessage: PropTypes.func
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->